tools/odbc/driver.cpp

```diff
@@ -126,65 +128,72 @@ SQLRETURN SQL_API SQLSetEnvAttr(SQLHENV environment_handle, SQLINTEGER attribute
 		case SQL_OV_ODBC2:
 			// TODO actually do something with this?
 			// auto version = (SQLINTEGER)(uintptr_t)value_ptr;
+			env->odbc_version = major_version;
```

Comment:
Is this still a TODO?

Reply:
Yes, now it's just being set and retrieved but never used.